ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot

```diff
@@ -77,7 +78,7 @@ Verify RHODS Accept Multiple Admin Groups And CRD Gets Updates
     Clear User Management Settings
     # Create a configmap and store both groups
     ${return_code}    ${cmd_output}=    Run And Return Rc And Output
-    ...    oc create configmap ${UPGRADE_CONFIG_MAP} -n ${UPGRADE_NS} --from-literal=adm_groups="['rhods-admins', 'rhods-users']" --from-literal=allwd_groups="['system:authenticated']"
+    ...    oc create configmap ${USERGROUPS_CONFIG_MAP} -n ${UPGRADE_NS} --from-literal=adm_groups="['rhods-admins', 'rhods-users']" --from-literal=allwd_groups="['system:authenticated']"
```

Comment:
Post-upgrade should be updated too, right? Including deletion of the config map after tests.